benchmarks/dbscan/ArborX_DBSCANVerification.hpp

```diff
@@ -17,6 +17,9 @@
 #include <ArborX_LinearBVH.hpp>
 
 #include <Kokkos_Core.hpp>
+#ifdef KOKKOS_VERSION >= 40200
+#include <Kokkos_Printf.hpp>
+#endif
```

Comment:
When/if we try to include less than `Kokkos_Core.hpp` we already have the minimal header required for `Kokkos::printf` in place.

Reply:
That does not apply to the other occurrences.  `Kokkos_Core.hpp` is needed in all examples.

As far as that particular header is concerned, it will be easy enough to figure out that header is needed if the refactor ever happens.

Reply:
I just removed it again.